### PR TITLE
Add prelim helm integration guide

### PR DESCRIPTION
**Pull request by efiacor**
Adding a placeholder for helm int docs and including a basic readme howto guide on using the existing flux kpt packages.
Also, some refactoring on the structure of the the docs repo.

### CODE REVIEW

**Comment by efiacor** [comment]
@liamfallon

File: user-guide/docs/helm/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Helm Integration in Nephio
+
+## Overview
+
+Helm integration support in Nephio is currently under investigation 
+and will progress over the coming releases.\
```
Comment (johnbelamaric) [inline user-guide/docs/helm/README.md:6]: Extraneous backslash

File: user-guide/docs/helm/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Although the direction from Nephio is to encourage the use of 
+[operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) 
+for deploying on Kubernetes, legacy helm support will be required for a number of releases.
+
+### Deployment options
+
+[Using Flux CD to deploy helm charts in Nephio](https://github.com/nephio-project/docs/blob/main/install-guide/docs/helm/flux-helm.md)
```
Comment (johnbelamaric) [inline user-guide/docs/helm/README.md:13]: Make this a bulleted list, and then add the other options under consideration (even though they will not have links). At the very least add the Operator SDK and a link to that.

I would also change the link text to just "Using Flux CD".

File: user-guide/docs/helm/flux-helm.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+# Deploying helm charts in Nephio using Flux CD
+
+[Flux CD](https://fluxcd.io/flux/use-cases/helm/) provides a set of kubernetes controllers to enable a GitOps driven deployment of helm charts, using relevant the flux Custom Resources.
```
Comment (johnbelamaric) [inline user-guide/docs/helm/flux-helm.md:3]: s/kubernetes/Kubernetes/

I would just end the sentence where the comma is.

**Review by johnbelamaric — CHANGES_REQUESTED** [review]
A few minor tweaks, and I haven't tried it, but looks good generally.

File: user-guide/docs/helm/flux-helm.md (hunk 3 above)
Comment (efiacor) [reply user-guide/docs/helm/flux-helm.md]: Done

File: user-guide/docs/helm/README.md (hunk 2 above)
Comment (efiacor) [reply user-guide/docs/helm/README.md]: Done

File: user-guide/docs/helm/README.md (hunk 1 above)
Comment (efiacor) [reply user-guide/docs/helm/README.md]: Removed

**Review by johnbelamaric — APPROVED** [review]
/approve
/lgtm

**Pull request merged**